app/src/main/java/com/kazufukurou/nanji/model/Language.kt

```diff
@@ -11,4 +11,5 @@ enum class Language(@StringRes val title: Int) {
   ko(R.string.languageKo),
   ru(R.string.languageRu),
   en(R.string.languageEn)
-}
\ No newline at end of file
+  cz(R.string.languageCz)
+}
```

Comment:
This won't compile because:
- missing `,` at the end of previous line
- Date/time display logic for Czech is not implemented [here](https://github.com/artyommironov/nanji/blob/df04e12f11c906a22e1ea9b0745fe186e7a83be4/app/src/main/java/com/kazufukurou/nanji/ui/WidgetPresenter.kt#L53).

`Language.cz` enum entry and `R.string.languageCz` strings should be added in a separate PR that adds custom display logic for Czech language (if it will be implemented).

Reply:
ok, I create separate PR for that